namespace-provisioner/use-case1.md

```diff
@@ -64,25 +64,57 @@ The [strict ScanPolicy](https://github.com/vmware-tanzu/application-accelerator-
 
```

Comment:
nit: Line 61 mentions ` notAllowedSeverities := [] ` but it has the whitespaces inside the code block so it looks kind of odd. Please remove the whitespaces and add them outside the code block, like this: 

"(...) is configured using the `notAllowedSeverities := []` part of the rego file section of ScanPolicy."

Reply:
updated the code to remove the extra whitespaces